app/config/Config.scala

```diff
@@ -28,6 +32,12 @@ object Config extends AwsInstanceTags {
 
   lazy val mediaAtomMakerUrl: String = s"https://video.$domain"
 
+  // TODO do this better!
+  lazy val mediaAtomMakerUrlForCode: String = stage match {
```

Comment:
Maybe not the best name as it will handle `PROD` as well, but it'll do for now 👀